pageserver/src/tenant/timeline/detach_ancestor.rs

```diff
@@ -537,13 +981,51 @@ pub(super) async fn complete(
     tenant: &Tenant,
     prepared: PreparedTimelineDetach,
     _ctx: &RequestContext,
-) -> Result<Vec<TimelineId>, anyhow::Error> {
+) -> Result<(Vec<TimelineId>, bool), anyhow::Error> {
```

Comment:
This method can no longer be called complete because it's doing `detach_reparent` and complete exists confusingly in `SharedState`.